platzky/engine.py

```
@@ -7,11 +7,12 @@
 from flask_babel import Babel
 
 from platzky.config import Config
+from platzky.db.db import DB
 from platzky.models import CmsModule
 
 
 class Engine(Flask):
-    def __init__(self, config: Config, db, import_name):
+    def __init__(self, config: Config, db: DB, import_name: str):
```

Ruff (0.14.10)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)